tests/phpunit/tests/db.php

```diff
@@ -1511,6 +1509,21 @@ public function test_charset_switched_to_utf8() {
 		$this->assertSame( 'utf8_general_ci', $result['collate'] );
 	}
 
+
+	/**
+	 * @ticket 56933
+	 *
+	 * @covers wpdb::prepare
+	 */
+	public function test_prepare_with_placeholders() {
+		global $wpdb;
+
+		$sql      = $wpdb->prepare( "'%%%s%%'", 'hello' );
+		$expected = '}hello{';
```

Comment:
Should we add some clarification here or in the PHPDoc? It's not immediately obvious that this is testing for the absence of single quotes (by using the adjacent curly braces that delimit that hex value).

Reply:
I think the test name change to be more specific and self-documenting.

Reply:
Looks good to me! 👍